aiida/backends/testbase.py

```diff
@@ -185,6 +185,22 @@ def assertClickResultNoException(self, cli_result):
         self.assertIsNone(cli_result.exception, ''.join(traceback.format_exception(*cli_result.exc_info)))
 
 
+class AiiDAPostgresTestCase(AiidaTestCase):
+
+    @classmethod
+    def setUpClass(cls, *args, **kwargs):
+        """Setup the PGTest postgres test cluster."""
+        super(AiiDAPostgresTestCase, cls).setUpClass(*args, **kwargs)
+        from pgtest.pgtest import PGTest
+        cls.pg_test = PGTest()
+
+    @classmethod
+    def tearDownClass(cls, *args, **kwargs):
+        """Close the PGTest postgres test cluster."""
+        super(AiiDAPostgresTestCase, cls).tearDownClass(*args, **kwargs)
+        cls.pg_test.close()
```

Comment:
this order is a bit inconsistent with the order in the setupclass.
I guess it doesn't matter since the AiidaTestCase does not interact with pgtest directly.

Still, we should probably consider the AiidaTestCase the "inner" thing, i.e. create the pgcluster first and delete it last